internal/app/connectconformance/test_case_library.go

```diff
@@ -177,18 +178,33 @@ func (lib *testCaseLibrary) expandCases(cfgCase configCase, namePrefix []string,
 		if testCase.Request.TestName == "" {
 			return fmt.Errorf("test case #%d: test case has no name", i+1)
 		}
-		if testCase.Request.Service == "" {
-			return fmt.Errorf("test case #%d: test %s has no service specified", i+1, testCase.Request.TestName)
-		}
-		if testCase.Request.Method == "" {
-			return fmt.Errorf("test case #%d: test %s has no method specified", i+1, testCase.Request.TestName)
-		}
 		if testCase.Request.StreamType == conformancev1.StreamType_STREAM_TYPE_UNSPECIFIED {
 			return fmt.Errorf("test case #%d: test %s has no stream type specified", i+1, testCase.Request.TestName)
 		}
 		if testCase.Request.StreamType != cfgCase.StreamType {
 			continue
 		}
+		// If service isn't specified, default to the Conformance Service.
+		if testCase.Request.GetService() == "" {
+			serviceName := conformancev1connect.ConformanceServiceName
+			testCase.Request.Service = &serviceName
+		}
+		// If method isn't specified, auto-populate based on stream type
+		if testCase.Request.GetMethod() == "" {
```

Comment:
Nit from above comment: we should only fill in a default value for the method if the service name is also blank.

We could even require that we never set a default on either unless both are blank. If we went that far, then the tests for "unimplemented" would have to specify both the service and method name, which I think might be okay and even best for clarity.